### PR TITLE
Adding ARG_COMMAND to pass a command as args

### PR DESCRIPTION
**Pull request by bjlee2000**

### CODE REVIEW

File: pytwis/pytwis_clt.py [hunk 1]
```diff
@@ -221,96 +222,100 @@ def pytwis_command_parser(raw_command):
...
+        arg_dict = parse.parse('{{{arg1}}} {{{arg2}}}'.format(arg1=CmdConstant.ARG_USERNAME,
                                                               arg2=CmdConstant.ARG_PASSWORD), args)
         if arg_dict is None:
             raise ValueError('{} has incorrect arguments'.format(CmdConstant.CMD_REGISTER))
         elif ' ' in arg_dict[CmdConstant.ARG_PASSWORD]:
             raise ValueError("password can't contain spaces")
 
-        print('{}: username = {}, password = {}'.format(CmdConstant.CMD_REGISTER, 
-                                                        arg_dict[CmdConstant.ARG_USERNAME], 
+        arg_dict = arg_dict.named
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: May I ask what "named" is?

File: pytwis/pytwis_clt.py [hunk 2]
```diff
@@ -593,28 +532,99 @@ def pytwis_cli():
 
     try:
         if len(args.socket) > 0:
-            twis = Pytwis(socket=args.socket, db=args.db, password=args.password)
+            pytwis = Pytwis(socket=args.socket, db=args.db, password=args.password)
         else:
-            twis = Pytwis(hostname=args.hostname, port=args.port, db=args.db, password=args.password)
+            pytwis = Pytwis(hostname=args.hostname, port=args.port, db=args.db, password=args.password)
+        return pytwis, prompt
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Where do we set "prompt"?

File: pytwis/pytwis_clt.py [hunk 3]
```diff
@@ -593,28 +532,99 @@ def pytwis_cli():
...
+    by the currently logged-in user.
+
+    * Exit the program:
+
+        127.0.0.1:6379> exit
+        127.0.0.1:6379> quit
+    '''
+    pytwis, prompt = get_pytwis(epilog)
+    if pytwis == None:
+        return
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: Return an error value like -1?

**Review by renweizhukov — CHANGES_REQUESTED** [review]
Could you please take a look at my comments? Thanks.

File: pytwis/pytwis_clt.py (hunk 2 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Line 520. 
    if len(args.socket) > 0:
        print('The input Redis server socket is {}'.format(args.socket))
        prompt = args.socket
    else:
        print('The input Redis server hostname is {}.'.format(args.hostname))
        print('The input Redis server port is {}.'.format(args.port))
        prompt = '{}:{}'.format(args.hostname, args.port)

File: pytwis/pytwis_clt.py (hunk 3 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: Do you want to have three returns? error, pytwis, prompt?

File: pytwis/pytwis_clt.py (hunk 1 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: parse doesn't return a dictionary. 
https://pypi.python.org/pypi/parse

Comment (renweizhukov) [reply pytwis/pytwis_clt.py]: Oh, I didn't know that. Thank you for answering my question!

File: pytwis/pytwis_clt.py [hunk 4]
```diff
@@ -319,20 +324,20 @@ def pytwis_command_parser(raw_command):
             args = splited_raw_command[1] 
             arg_dict = parse.parse('{{{arg1}}} {{{arg2}:d}}'.format(arg1=CmdConstant.ARG_USERNAME,
                                                                     arg2=CmdConstant.ARG_MAX_TWEETS), 
-                                   args)
+                                                                    args)
             if arg_dict is None:
```
Comment (renweizhukov) [inline pytwis/pytwis_clt.py]: If arg_dict is not None, do we need to set "arg_dict = arg_dict.named"?

File: pytwis/pytwis_clt.py (hunk 2 above)
Comment (renweizhukov) [reply pytwis/pytwis_clt.py]: Sorry. My bad. I just looked at the diff part and failed to expand the common part. Now I see it.

File: pytwis/pytwis_clt.py (hunk 3 above)
Comment (renweizhukov) [reply pytwis/pytwis_clt.py]: No, I mean that we return -1 if pytwis == None.

**Review by renweizhukov — COMMENTED** [review]
Thank you for the response.

File: pytwis/pytwis_clt.py (hunk 3 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: I don't think that's good idea. If the first return value is the instance of Pytwis, then it must be None for error case, not an integer value, -1.

File: pytwis/pytwis_clt.py (hunk 4 above)
Comment (bjlee2000) [reply pytwis/pytwis_clt.py]: I changed the code to check if arg_dict is parse.Result and do the conversion at the end.

File: pytwis/pytwis_clt.py (hunk 3 above)
Comment (renweizhukov) [reply pytwis/pytwis_clt.py]: Clarified my point offline.

**Review by renweizhukov — APPROVED** [review]

**Pull request merged**